src/AutorestSwift/View Models/KeyValueViewModel.swift

```diff
@@ -158,11 +171,11 @@ struct KeyValueViewModel {
         - Parameter key: Key String in the Key value pair
         - Parameter value: the value string
      */
-    init(key: String, value: String) {
+    init(key: String, value: String, optional: Bool = false, implmentation: Implmentation = .options) {
```

Comment:
Type here: implmentation -> implementation

Reply:
Remove in new iteration.